homeassistant/components/stream/__init__.py

```diff
@@ -154,7 +156,8 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     hass.data[DOMAIN] = {}
     hass.data[DOMAIN][ATTR_ENDPOINTS] = {}
     hass.data[DOMAIN][ATTR_STREAMS] = []
-    if (conf := config.get(DOMAIN)) and conf[CONF_LL_HLS]:
+    conf = DOMAIN_SCHEMA(config.get(DOMAIN, {}))
```

Comment:
I needed to make this change to get the default value of True to apply when no `stream` is present in the config. Not sure if there is a better way to do this, but then it makes it safe on the next line to always have an `ll_hls` value.

Reply:
This is one approach. Alternative approach is to not set default values in voluptuous schema but apply when you get the value from the schema. Both are fine.